### PR TITLE
Use BaseJdbcConnectorTest for Phoenix connectors

### PR DESCRIPTION
**Pull request by kokosing**
Use BaseJdbcConnectorTest for Phoenix connectors

### CODE REVIEW

File: plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java [hunk 1]
```diff
@@ -54,6 +56,125 @@ public void destroy()
...
+    @Override
+    protected TestTable createTableWithDefaultColumns()
+    {
+        throw new SkipException("Phoenix connector does not support column default values");
+    }
+
+    @Override
+    public void testRenameTable()
+    {
+        // Phoenix does not support renaming tables
```
Comment (ssheikin) [inline plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java:98]: `SkipException`?

File: plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java [hunk 2]
```diff
@@ -54,6 +56,125 @@ public void destroy()
...
+    }
+
+    @Override
+    public void testRenameColumn()
+    {
+        // Phoenix does not support renaming columns
```
Comment (ssheikin) [inline plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java:104]: `SkipException`?

**Review by ssheikin — APPROVED** [review]

File: plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java (hunk 1 above)
Comment (losipiuk) [reply plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java:98]: Actially it should be `assertThatThrownBy(super.testRenameTable()); throw new SkipException()`
But it is preexistent.

**Review by losipiuk — APPROVED** [review]

File: plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java [hunk 3]
```diff
@@ -54,6 +56,129 @@ public void destroy()
...
+        throw new SkipException("Phoenix connector does not support column default values");
+    }
+
+    @Override
+    public void testRenameTable()
+    {
+        assertThatThrownBy(super::testRenameTable)
+                // TODO (https://github.com/trinodb/trino/issues/7204) support table rename in Phoenix
+                .hasMessageContaining("Syntax error. Encountered \"RENAME\"");
+    }
```
Comment (findepi) [inline plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java:101]: `throw new SkipException`, the test should not show up as passing

(same below)

Comment (kokosing) [reply plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java:101]: Thanks. https://github.com/trinodb/trino/pull/7214

**Pull request merged**